src/lavatory/utils/get_artifactory_info.py

```diff
@@ -0,0 +1,29 @@
+"""Helper method for getting artifactory information."""
+import logging
+
+from .artifactory import Artifactory
+
+
+def get_artifactory_info(repo_name=None, repo_type='local'):
+    """Get storage info from Artifactory.
+
+    Args:
+        repo_name (tuple, optional): Name of artifactory repo.
```

Comment:
`repo_names` might be better if this is expected to be a tuple with multiple values.

Reply:
Changed to plural.